src/pages/ListCandidatesVacancies.jsx

```diff
@@ -11,6 +11,41 @@ import { SuccessfulApplicant } from '../components/SuccessfulApplicant';
 
 import '../style/listCandidatesVacancies.scss';

```

Comment:
Are these styles importants?

Reply:
I think we can change that when working on this view. I made changes in two components used by this view and the layout needs to be fixed